apiserver/undertaker/state.go

```diff
@@ -31,62 +30,18 @@ type State interface {
 	// collections.
 	RemoveAllModelDocs() error
 
-	// AllMachines returns all machines in the model ordered by id.
-	AllMachines() ([]Machine, error)
-
-	// AllApplications returns all deployed services in the model.
-	AllApplications() ([]Service, error)
-
 	// ModelConfig retrieves the model configuration.
 	ModelConfig() (*config.Config, error)
+
+	// WatchModelEntitiesReferences gets a watcher capable of monitoring
```

Comment:
Changed :D nice pick-up! I was, of course thinking of model entities as a concept.